### PR TITLE
Ref manager fixes

### PR DESCRIPTION
**Pull request by lightninglu10**
Resolves issues listed here:

Screenshot 2023-12-05 at 11 27 47 AM

### CODE REVIEW

File: components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx [hunk 1]
```diff
@@ -117,7 +114,12 @@ export default function ReferenceManualUploadDrawer(): ReactElement {
             required: referenceSchemaValueSet.required,
           });
         };
-        if (schemaField === "pdf_url" || schemaField === "custom") {
+        console.log(schemaField);
```
Comment (manveerxyz) [inline components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx:117]: nit: should probably remove the console.log

File: components/ReferenceManager/references/reference_uploader/reference_upload_utils.ts [hunk 2]
```diff
@@ -133,6 +133,10 @@ export const handleSubmit = ({
       ...referenceSchemaValueSet.schema,
       author: formattedCreators,
       issued: formattedCSLDate,
+      custom: {
+        ...referenceSchemaValueSet.custom,
+        attachment: referenceSchemaValueSet.signedUrl,
```
Comment (manveerxyz) [inline components/ReferenceManager/references/reference_uploader/reference_upload_utils.ts:138]: what does the `signedUrl` do?

**Review by manveerxyz — COMMENTED** [review]
Lots of good fixes. Reference Type dropdown looks great as well I like it.

I tried renaming my folder in staging from "My Library" to "Default", but it didn't rename the header — even on page refresh, but it does rename the folder in the sidebar.
Screenshot 2023-12-05 at 12 18 48 PM

**Comment by lightninglu10** [comment]
> Lots of good fixes. Reference Type dropdown looks great as well I like it.
> 
> I tried renaming my folder in staging from "My Library" to "Default", but it didn't rename the header — even on page refresh, but it does rename the folder in the sidebar. Screenshot 2023-12-05 at 12 18 48 PM

You need the backend branch as well, that's an update from the backend

File: components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx (hunk 1 above)
Comment (lightninglu10) [reply components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx:117]: Ah yes, def should remove

**Comment by manveerxyz** [comment]
> > Lots of good fixes. Reference Type dropdown looks great as well I like it.
> > I tried renaming my folder in staging from "My Library" to "Default", but it didn't rename the header — even on page refresh, but it does rename the folder in the sidebar. Screenshot 2023-12-05 at 12 18 48 PM
> 
> You need the backend branch as well, that's an update from the backend

ah okay my bad haha

**Pull request merged**